srcDoc/tutorials/performance_validation.rst

```diff
@@ -0,0 +1,642 @@
+        512,725,130.1,551,6414,716,799,1002,5175,6414
+        1024,868,366.8,676,36814,851,938,1133,6913,36814
+        2048,1095,162.8,879,6341,1088,1177,1433,6341,6341
+        4096,1309,453.6,1083,38591,1292,1379,1643,38591,38591
+        8192,1666,167.7,1349,6790,1651,1769,2032,6790,6790
+        16384,2416,628.4,2146,39850,2396,2516,2844,39850,39850
+        32768,4046,246.8,3732,8894,4042,4161,4594,8894,8894
+        63000,6909,176.8,6564,9529,6896,7102,7368,9529,9529
+
+
+Understanding the Results
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Lets go first with the throughput results and plot all the different tests together:
+
+.. image:: performance_validation_files/Throughput_lineal.svg
+
+The first think we see is that at 5KB we are already close to saturate the
+network in all cases, which is something really good to see, but lets focus
+in the behavior for smaller samples. Lets plot the same results with a
+logarithmic scale:
+
+.. image:: performance_validation_files/Throughput_log.svg
+
+Now we can extract more information about the graphs:
+
+1. If we take out the test where we make use of *batching* we can see that using
+   Raw Transport (plain sockets) gives us the best performance.
+
+2. **Connext DDS Pro** and **Connext DDS Micro** behave in a really similar way,
+   being the latter slighly better.
+
+3. The use of *batching* really makes a difference for small samples sizes.
+
+4. After 5kB we can consider that all the tests are able to reach more than a 95%
+   of the network utilization, which all what these boards can offer.
+
+By what we state in 1, one might wonder why not using plain sockets for our communications,
+why having use a middleware for this. However, at this point is when we have to remember
+that when testing with *Plain Sockets*, we had nothing: We didn't have a discovery
+mechanism (we had to specify the peers by hand), we didn't have any reliability and samples
+would not get repaired when lost. In fact we didn't have any *QoS* setting at all.
+Also we would not be able to make use of any multicast feature, so scalability would have been
+an issue as well.
+
+by using **RTI Connext DDS** you are adding a discovery mechanism, a reliability mechanism,
+the option of tuning the *QoS* of the system, etc. Lastly, you have to remember what we
+stated in 4 and 3: The advantage of *Plain Sockets* is only noticeable when the data length
+is quite small, and even in those cases, by using certain features, **RTI Connext DDS** can
+keep up, or even improve the performance provided by *Raw Sockets*.
+
+Another important topic is if we should choose **RTI Connext DDS Micro** instead of
+**Professional** based on the performance you want to achieve. Although it is correct that
+the former will achieve better performance for simple scenarios like this
+one, **Professional** has the advantage of all the features it has (like *batching* or
+*Content-Filtered Topics*), so the answer to this question is that in general the
```

Comment:
**Professional** offers more features than Micro (like *batching* ...). On the other side, Micro is ideal to run in resource constraint devices where **Professional** may not fit. 

I would remove the setence:

in general the performance difference ... It does not sound right

Reply:
Done.